src/BlazorUI/Bit.BlazorUI/Components/Inputs/ChoiceGroup/BitChoiceGroup.scss

```
@@ -137,14 +195,51 @@
+// Every item label carries .bit-chg-itl and the tile layout only adds .bit-chg-ili on top of it, so the
+// single rule here covers both the plain and the tile item without them stacking two rings on each other.
+.bit-chg-inp:focus-visible + .bit-chg-itl {
+    @include focus-ring(var(--bit-chg-clr-focus));
+}
+
 .bit-chg-itl {
     display: flex;
     cursor: pointer;
     gap: spacing(0.75);
     align-items: center;
+    // Matches the radius the tile layout sets below, so the focus ring (a box-shadow, which follows the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)